backend/tournesol/views/exports.py

```diff
@@ -49,6 +86,18 @@ def get(self, request):
         return response
 
 
+class ExportPublicComparisonsView(APIView):
+    @extend_schema(
+        description="Download public data in .zip file",
```

Comment:
I see this description is incorrect, as is the description in `ExportComparisonsView` so that's mnot on you. The file extension is csv.

Reply:
Should I change the description into `.csv` in `ExportComparisonsView` as well?

Reply:
Yes please. That would be great